main.html

```diff
@@ -0,0 +1,189 @@
+        <li><a href="about.html">About</a></li>
+        <li><a href="main.html">Home</a></li>
+        <li>Program</li>
+        <li>Join</li>
+        <li>News</li>
+ </ul>
+
+</div>
+        </article>
+
+    </section>
+
+    <main>
+        
+<div class="pg1">
+    <h2 class="theme">
+        "Youth For Christ"
+     </h2>
+     <h1 class="ch-name">Winners Chapel Youth Conference 2022.</h1>
+    <div class="moment">
+         <p class="prg">in His presnce there is fullness of joy, experience God in another demention come youth 2022 conference</p>
+    </div>
+     <div class="info">
+        <h6 class="date">22.05.2022(THUR)~23(FRI)</h6>
+        <p class="venue">@ Church Auditorium, Lagos.</p>
+    </div>
+ </div>
+
+ <section class="prog-card">
+    <h5 class="h5">Main program</h5>
+    <hr class="hr2">
+    <div class="card-section">
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+                    <li><img class="img-w" src="./images/lecture.png" alt=""></li>
+                   <li class="lpd" >Word</li>
+                 <li  class="msg">Out pouring of the spirit of God through His word by the man of God as He give utterances.</li>
+           </ul>
+        </div>
+</div>
+        
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w" src="./images/lecture.png" alt=""></li>
+            <li class="lpd2">Prayers</li>
+            <li><p class="msg1">Its a time to draw down the presence of God and seeking His presence through prayers.</p></li>
+        </ul>
+    </div>
+</div>
+
+        <div  class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w" src="./images/lecture.png" alt=""></li>
+            <li class="lpd3">Worship</li>
+            <li><p class="msg2">praise and worhip secton to bring down the glory of God with annoited gospel musician.
+            </p>
+        </li>
+        </ul>
+    </div>
+</div>
+
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w" src="./images/lecture.png" alt=""></li>
+            <li class="lpd5">Discusion</li>
+            <li><p class="msg3">Building strong and vibrant youth for christ through moral building and responsible youth in the society</p></li>
+        </ul>
+    </div>
+</div>
+
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w"   src="./images/lecture.png" alt=""></li>
+            <li class="lpd4">Fasting</li>
+            <li><p class="msg4">This event will take place on the last day of worship.</p></li>
+        </ul>
+    </div>
+</div>
+    </div>
+    <div class="join">
+    <p class="us">Worship With Us</p>
+</div>
+
+<div class="join2">
+    <p><a href=""> See More Program</a></p>
```

Comment:
- [ ] This link should have the same color and font from the design

Reply:
corrected